LambdaEq.dfy(10,11): Error: assertion violation
Execution trace:
    (0,0): anon0
    LambdaEq.dfy(6,24): anon7_Else
    LambdaEq.dfy(7,24): anon8_Else
    LambdaEq.dfy(8,24): anon9_Else
LambdaEq.dfy(17,11): Error: assertion violation
Execution trace:
    (0,0): anon0
    LambdaEq.dfy(15,24): anon5_Else
    LambdaEq.dfy(16,24): anon6_Else

Dafny program verifier finished with 0 verified, 2 errors
